src/web/app/pages-instructor/instructor-search-page/instructor-search-bar/instructor-search-bar.component.html

```diff
@@ -9,11 +9,33 @@
     </div>
     <div class="form-group">
       <div class="input-group">
-        <input type="text" class="form-control" [(ngModel)]="searchKey" (keyup.enter)="search()"
-            placeholder="Your search keyword">
+        <input type="text" class="form-control" [ngModel]="searchParams.searchKey"
+          (ngModelChange)="triggerSearchParamsChangeEvent({
```

Comment:
Try search `triggerModelChange` method in the code base and you will find that you don't need to write this so verbosely.